lib/menu.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'dish'
+class Menu
+  attr_reader :menu_list
+  def initialize (menu_list=Hash.new)
```

Comment:
Style/EmptyLiteral: Use hash literal `{}` instead of `Hash.new`. (https://github.com/bbatsov/ruby-style-guide#literal-array-hash)<br>Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)